commit f78a9269228f2f48e8eb496e341ef990db39c6c3
Author: Alexis Ballier <aballier@gentoo.org>
Date:   Fri Sep 21 08:22:55 2012 -0300

    cairo-sphinx: protect -ldl link with CAIRO_HAS_DL like in cairo-trace.
    
    Signed-off-by: Uli Schlachter <psychon@znc.in>

diff --git a/util/cairo-sphinx/Makefile.am b/util/cairo-sphinx/Makefile.am
index b25bd23..10bc10c 100644
--- a/util/cairo-sphinx/Makefile.am
+++ b/util/cairo-sphinx/Makefile.am
@@ -12,7 +12,9 @@ cairo_sphinx_la_SOURCES = fdr.c
 cairo_sphinx_la_CPPFLAGS = $(AM_CPPFLAGS)
 cairo_sphinx_la_CFLAGS = $(CAIRO_CFLAGS)
 cairo_sphinx_la_LDFLAGS = -module -no-undefined
+if CAIRO_HAS_DL
 cairo_sphinx_la_LIBADD = -ldl
+endif
 
 cairo_sphinx_SOURCES = sphinx.c
 cairo_sphinx_CPPFLAGS = $(AM_CPPFLAGS) -DLIBDIR="\"$(cairolibdir)\""
